addon/components/new-bourbon-select-field-option.js

```diff
@@ -1,14 +1,20 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 import layout from '../templates/components/new-bourbon-select-field-option';
 
 export default Component.extend({
   layout,
   tagName: 'li',
   classNames: ['NewBourbonSelectField-option', 'BourbonSelectField-option'],
   classNameBindings: ['selected:btw-bg-concrete', 'active:btw-bg-concrete'],
-  attributeBindings: ['disabled', 'active', 'selected:selected', 'tabindex', 'role', 'selected:aria-selected'],
+  attributeBindings: ['isDisabled:disabled', 'active', 'selected:selected', 'tabindex', 'role', 'selected:aria-selected'],
   label: null,
-  disabled: false,
   active: false,
-  selected: false
+  selected: false,
+  isDisabled: computed('disabled', function () {
+    let disabled = this.get('disabled');
+    // returning false just sets it to the string "false"
+    // returning undefined makes it just ignore it.
+    return disabled ? disabled : undefined;
```

Comment:
i think this property can be simplified as `return this.get('disabled') || undefined`

Reply:
good call.  let me do that.